### ISSUE TITLE
Create a Slack welcome app

### ISSUE THREAD

**Issue by egiurleo** [wnbrb/wnb-rb-site#171]
**Is your feature request related to a problem? Please describe.**
Currently, we don't do a good job of welcoming new members to the WNB.rb Slack community. There are too many new people joining to greet them individually, which means we miss out on a crucial opportunity to make new members feel welcome and supported.

**Describe the solution you'd like**
I have written a [document introducing new members to the WNB.rb Slack workspace](https://docs.google.com/document/d/17eRDei9ElOX1-kDAjOhEt4NNDrulDkGHehSgRrKgXgc/edit#). I would like this document to be automatically sent to anyone who joins the WNB.rb Slack workspace by a Slack bot.

**Why don't we just use Slack Workflows?**
Workflows are a paid feature and it's not financially feasible for WNB.rb to pay for Slack right now.

**Desired Approach**
- I have already created a new Slack app that we can use to send messages to users. Please reach out to me on WNB.rb Slack so I can share the credentials with you.
- Follow this tutorial for using the [Slack Events API](https://api.slack.com/apis/connections/events-api)
  -  Basically, you'll create a new endpoint that the Slack app will ping every time a new user joins our workspace; you'll have to follow the guidelines in the tutorial above to make sure that endpoint responds appropriately to Slack event requests.
  - The event will trigger a message from the Slack app to the new user (probably using the [slack-ruby-client](https://github.com/slack-ruby/slack-ruby-client#send-messages))

This is a fairly significant piece of work that will require some previous knowledge of Rails API endpoints, requests, and responses.

---
**Current Status:**
This is a proposed feature that has been in the idea stage for 3 years. While it’s not a critical feature, it should be discussed with the website team to determine whether it should remain open.

**Comment by jrussell416** [comment]
**Issue Review**

**Recommendation:** Discuss Further

**Reason:** This feature is in the idea phase. Further discussion should take place to discuss if this should be implemented and if so, how to proceed. 
The welcome message is not critical but could serve as a nice welcome to the group and direct new members to key channels.

**Related Issues/PRs:** [About Us Issue](https://github.com/wnbrb/wnb-rb-site/issues/153) Welcome document linked above contains text that could potentially by used in an "About Us" section 

**Next Steps:** Discuss with Sarah and issue authors

**Comment by sarah-fflow** [comment]
Superceded by our migration from slack to discord, no longer required.
